pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_pantheon_yaml_file.adoc

```diff
@@ -1,19 +1,19 @@
 [id='pantheon-yaml-file_{context}']
 
-= The Pantheon configuration file
+= The {ProductShortName} configuration file
 
-You need to create a configuration file, named `pantheon2.yml`, in the root directory of your repository, to enable a Git repository as a source that Pantheon can use to import content. This yaml file provides necessary information that Pantheon uses to interact with files in the repository.
+You need to create a configuration file, named `pantheon2.yml`, in the root directory of your repository, to enable a Git repository as a source that {ProductShortName} can use to import content. This yaml file provides necessary information that {ProductShortName} uses to interact with files in the repository.
 
 The configuration file enables the following:
 
-* Importing content from a Git repository into Pantheon
-* Uploading modules into Pantheon using the Uploader tool
+* Importing content from a Git repository into {ProductShortName}
+* Uploading modules into {ProductShortName} using the Uploader tool
 
 The configuration file defines the following:
 
-`server`:: The URL of the Pantheon server.
-`repository`:: A unique ID that Pantheon uses to reference the URL of the repository. You specify the actual URL of the repository when you synchronize the repository the first time.
-`modules`:: Patterns for modules that show up in the search results on Pantheon.
+`server`:: The URL of the {ProductShortName} server.
+`repository`:: A unique ID that {ProductShortName} uses to reference the URL of the repository. You specify the actual URL of the repository when you synchronize the repository the first time.
```

Comment:
Actually, this has changed a bit. There is no longer a single `repository` entry, but rather a multi-valued `repositories` structure that has child members of `name`, `modules`, `resources`, and now additionally `attributes` which is a path to the repository-wide attributes file that your documents may depend on.

Reply:
@benradey Can you paste the code verbatim into a comment? Or an example?

Reply:
```
server: http://localhost:8080
repositories:
  - name: pantheonSampleRepo
    attributes: path/to/attribute.adoc

    modules:
      - master.adoc
      - modules/*.adoc

    resources:
      - shared/legal.adoc
      - shared/foreword.adoc
      - resources/*
```

Reply:
@benradey I have some questions:

1. Why is the name of the structure in the plural (repositories)? Can you have more than one repository?
1. There is a note in lines 29-30 that says:
  `## Note: Due to yaml syntax, any filepaths that start with a wildcard must be surrounded in quotes like so:`
`# modules:`
`#  - '*.adoc'`
 However in your example, you don't include the quotes around the wildcards. Should I remove that note?

Reply:
1. Sort of, yes. Because the attributes file is a repository-wide setting, if you have multiple attributes files that you build documents with, then you will be setting up multiple pantheon repositories. This isn't officially supported yet, but it's on the radar, so we are setting the stage for that now.
2. The quotes are only necessary if the line _starts_ with the wildcard. If there's a wildcard in the middle of the entry, then the quotes aren't necessary.
3. What I pasted is the entirety of what the uploader actually reads. I stripped out the surrounding comments. So assuming your question was a question and not a request that I misunderstood, the answer is "yes, that is the entire file."